sdk/storage_blobs/src/options/condition_tags.rs

```diff
@@ -0,0 +1,26 @@
+use azure_core::headers::{Header, HeaderName, HeaderValue};
+
+#[derive(Debug, Clone)]
+pub struct IfTagsCondition(String);
```

Comment:
We've been using `Cow<'static, str>` with great success. Should we consider that here?

Reply:
Opened PR to do that